controller/todos.go

```diff
@@ -0,0 +1,47 @@
+package controller
+
+import (
+	"github.com/gin-gonic/gin"
+	"godemo/database"
+	"godemo/model"
+	"net/http"
+)
+
+type todos struct{}
+
+// TODO関連の処理を行うコントローラ
+var Todos todos = todos{}
```

Comment:
should omit type todos from declaration of var Todos; it will be inferred from the right-hand side